src/main/scripts/validate_deployment.groovy

```diff
@@ -59,7 +74,40 @@ println('command:')
 println(command.join(' '))
 println('')
 
-def process = command.execute()
+def exitCode = 0
```

Comment:
In the future, I suggest you look into CommandHelper that is part of the [groovy-plugin-utils library](https://github.com/IBM-UrbanCode/groovy-plugin-utils). It will help eliminate nearly all of the repeated ProcessBuilder code. With CommandHelper, the code would look something like...

CommandHelper ch = new CommandHelper(new File(WORKING_DIRECTORY_PATH or just "."));
// Construct command line call as String[]
ch.addEnvironmentVariable("LIBPATH", LIBPATH);
// etc
int exitCode = ch.runCommand(commandArray);
// Command output will be printed auto-magically

Hope that helps for the future. Nearly all of our supported plugins use some form of command helper.